trikot-viewmodels-declarative-flow/swift/swiftui/ViewModifiers/VMDModifier.swift

```diff
@@ -0,0 +1,29 @@
+import SwiftUI
+import TRIKOT_FRAMEWORK_NAME
+
+struct VmdModifier: ViewModifier {
```

Comment:
I think separating each operation into separate modifiers might be a better approach.

Reply:
As we talked earlier, the extension is public so it can be called from outside, the struct really doesn't need to.
Separating each operation would result to duplicate the calls everywhere in every component. Changing how they are created and re-using the modifiers might take more time to think through so this can come in another PR if some SwiftUI kingpin wants to tackle that.